src/platform/plugins/shared/dashboard/public/dashboard_api/search_sessions/new_session.ts

```diff
@@ -61,7 +61,7 @@ export function newSession$(api: unknown) {
   }
 
   if (apiPublishesReload(api)) {
-    observables.push(api.reload$);
+    observables.push(api.reload$.pipe(startWith(undefined)));
```

Comment:
This is required because the `combineLatest` on line 67 wasn't getting called. This caused the Controls to not create a new search session ID, and therefore not work until after refresh was pressed once.

Reply:
Good call, will do!